documentation/project-docs/SDK-PR-guide.md

```diff
@@ -0,0 +1,56 @@
+These quarterly releases should avoid breaking changes if at all possible. Breaking changes instead should be included in the major releases and be tied to customer TargetFramework updates when possible.
+
+Minor releases lock down after preview 3 for each release.  Preview 3 is typically 1 month prior to GA and locks down by the end of the month before the preview release.
+
+#### Transition from minor release to servicing
+After the minor release locks down, it transitions into servicing mode. This typically happens around the preview 3 release to allow time for any late public changes we want feedback on. The next release after preview 3 is typically GA and done as a servicing release. The following steps are done when transitioning from minor release previews to servicing
+- [ ] Follow the new branch check list for creating a new minor release (steps listed on internal OneNote)
+- [ ] Set stabilize branding to true
+- [ ] Enable internal runtime darc subscriptions and trigger them
+
+### Servicing releases
+The .NET SDK has monthly servicing releases aligning with the .NET Runtime servicing releases. These are for top fixes and security updates only to limit risk.
+Any servicing release is open for checkins from the day the [branding PRs](https://github.com/dotnet/sdk/pulls?q=is%3Apr+branding) are merged (~1st of each month) and when code complete is (typically two weeks later).
+The servicing branches are locked from the time of code complete to the next branding in case we need to respin any monthly release. Final signoffs are typically in the last week of each month.
+
+### Schedule
+| Release Type | Frequency    | Lockdown Release  | Branch Open | Lockdown Date (estimate) |
+| -------------|--------------|-------------------|-------------|--------------------------|
+| Major        | Yearly (Nov) | RC2               | ~August     | Mid-September            |
+| Minor        | Quarterly    | Preview 3         | ~Prior release Preview 3 date | End of the month prior to Preview 3 (~7 weeks prior to release) |
+| Servicing    | Monthly      | N/A               | After branding, ~1st of the month | Third Tuesday of prior month (signoff is ~28th of each month) |
+
+### Tactics approval
+Even releases that are in lockdown can still take changes as long as they are approved and the final build isn't complete. To bring a change through tactics, mark it with the label servicing-consider and update the description to include 5 sections (Description, Customer Impact, Regression?, Risk, Testing). See previously approved bugs for examples by looking for the [servicing-approved](https://github.com/dotnet/sdk/pulls?q=is%3Apr+label%3AServicing-approved+is%3Aclosed) label
+
+## External contributions
+External contributions are encouraged and welcome. There are so many teams working in this repo that it can be hard to track.
```

Comment:
```suggestion
External contributions are encouraged and welcome. There are so many teams working in this repo that it can be hard to track. Contributors looking to learn more about how to contribute should check out the [Developer Guide](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/developer-guide.md).
```